### ISSUE TITLE
Alt text on site logo

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#179]
I will be adding some issues regarding accessibility. I believe that all education websites have to be fully accessible by Jan. 1, 2018. We purchased an accessibility scanner (and I also use the [WAVE accessibility tool](http://wave.webaim.org/report#/www.jccmi.edu)) and it is coming up with some issues. The first one is that the main logo on the site is missing an alt text.

**Comment by BigActual** [comment]
Hi Ashley, would you like to run an audit by a trained professional?

**Comment by banksashleym** [comment]
Hi Steve,
I use the WAVE tool and we also purchased an online auditing tool called [dynomapper](https://dynomapper.com/). But I think when it comes to accessibility you can never be too safe! Especially since we are under deadline. Will this additional resource be an extra cost?

I know one of the big issues that I am tackling is getting alt text on all of the images. But I would love to hear feedback on anything else we could be doing to eliminate the risk of being out of compliance.

**Comment by BigActual** [comment]
Yeah, it would likely be a few thousand dollars to have it done. The outcome though is a truly detailed plan and guaranteed compliance.

**Comment by banksashleym** [comment]
Is there a tool/resource you recommend?

**Comment by BigActual** [comment]
I've reached out to Joe Dolson. He's always my first pick. Lot's of experience. I'll get a sense of what other colleges are doing and let you know what comes of the conversation.

**Comment by BigActual** [comment]
Here's the reply from Joe. His pricing will be indicative of other professionals in the community:

> There's no real "fixed" cost to an audit, as it depends enormously on scope. Assuming an audit based on the WCAG 2.0 standards at level AA and a moderate-sized website (150-200 pages, miscellaneous complex interactive features, etc.), you can expect a starting cost somewhere around $10000-$12000. That assumes a detail-oriented audit that explicitly looks at every single page on the site and reviews the entire scope of the site. 

> It can be cheaper doing more of a template-based audit. That scope would involve reviewing a representative example of every type of page template in use plus any complex interactive features. This doesn't guarantee that every issue will be caught, as content is a major factor in issues - but it can considerably reduce the amount of time spent on the review. 

> Only with the first type of review, however, can I provide any kind of documentation indicating that the site has been fully audited. 

> But ultimately, a quote *must* be based on some time spent looking at the actual site - it depends hugely on the complexity of the site, what kinds of features are present, how large the site is, etc.

Best,
Joe

**Comment by banksashleym** [comment]
Thanks! I will run this by Dotty, but I'm going to be that we don't have that kind of money in the web budget for this year. I will let you know if she would like to proceed with this.

Thanks!
